src/mg/PAMI/Client/Impl/ClientImpl.php

```diff
@@ -185,9 +189,7 @@ public function open()
 	    @stream_set_blocking($this->_socket, 0);
 	    $this->_currentProcessingMessage = '';
 	    //register_tick_function(array($this, 'process'));
-	    if ($this->_logger->isDebugEnabled()) {
```

Comment:
hmm.. well there *is* a lot of debug output, and we can save some cpu cycles right there, since we can't decouple the logging from the main loop.